builder/frameworks/fsp.py

```diff
@@ -0,0 +1,103 @@
+from os.path import isdir, join
+
+from SCons.Script import DefaultEnvironment
+
+env = DefaultEnvironment()
+platform = env.PioPlatform()
+board = env.BoardConfig()
+
+env.SConscript("_bare.py")
+
+FRAMEWORK_DIR = platform.get_package_dir("framework-renesas-fsp")
+assert isdir(FRAMEWORK_DIR)
+
+env.Append(
+    CFLAGS=[
+        "-std=gnu11"
+    ],
+
+    CPPDEFINES=[
+        "_RENESAS_RA_"
+    ],
+
+    CXXFLAGS=[
+        "-std=gnu++17",
+        "-fno-use-cxa-atexit"
+    ],
+
+    CPPPATH=[
+        join(FRAMEWORK_DIR, "fsp", "inc"),
+        join(FRAMEWORK_DIR, "fsp", "inc", "api"),
+        join(FRAMEWORK_DIR, "fsp", "inc", "instances"),
+        join(FRAMEWORK_DIR, "fsp", "src"),
+        join(FRAMEWORK_DIR, "fsp", "src", "r_sce"),
+        join(FRAMEWORK_DIR, "fsp", "src", "r_sce", "common"),
+        join(FRAMEWORK_DIR, "fsp", "inc", "arm", "CMSIS_5", "CMSIS", "Core", "Include"),
+        join(FRAMEWORK_DIR, "variants", board.get("build.variant")),
+        join(FRAMEWORK_DIR, "variants", board.get("build.variant"), "includes", "ra_gen"),
+        join(FRAMEWORK_DIR, "variants", board.get("build.variant"), "includes", "ra_cfg", "fsp_cfg"),
+        join(FRAMEWORK_DIR, "variants", board.get("build.variant"), "includes", "ra_cfg", "fsp_cfg", "bsp"),
+        join(FRAMEWORK_DIR, "variants", board.get("build.variant"), "tmp_gen_c_files")
```

Comment:
I see a lot of `board.get("build.variant")` calls, does it make sense to extract this value in a variable?

Reply:
Refactored into `varirant` variable in https://github.com/platformio/platform-renesas-ra/pull/7/commits/170b200dc9b2bebbeb37b29fad9b4bcb1abe94bd